pyOlog/ophyd_tools.py

```diff
@@ -446,10 +446,12 @@ def _print_pos(positioners, file=sys.stdout):
             continue
         if v is not None:
             try:
-                prec = p.precision
-            except (AttributeError, DisconnectedError):
-                prec = FMT_PREC
-            value = np.round(v, decimals=prec)
+                value = np.round(v, decimals=prec)
+            except TypeError:
+                value = v
+            except (AttributeError,DisconnectedError):
```

Comment:
Do we care about PEP8 in this project? I mean missed blank space.

Reply:
I will fix this, just waiting on some feedback from Tom on his comment